scripts/newinstall.sh

```diff
@@ -165,7 +165,7 @@ fi
 ##########	Test/warn about Python versions, offer to get miniconda2 if too old
 
 if true; then
-	PYVEROK=$(python -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
+	PYVEROK=$($PYTHON -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
```

Comment:
The relevant text up above indicating what we are setting `$PYTHON` to is:
```
# Use system python to bootstrap unless otherwise specified
```
Which implies to the naive user that this Python is only being used to bootstrap the system but will not be used when code is being run after the install. It seems that this python turns out to be the EUPS python because anaconda is installed after EUPS is configured. If it really is the case that this is purely a bootstrap python then is this check for python 2.7 really relevant now that EUPS works on python 3? If this python check is purely because we are playing it safe because people may have used an older EUPS then add a comment before this check indicating that this is purely defensive for older EUPS users.